src/assets/badges/svgXMLs/ceilingFans.ts

```diff
@@ -1,20 +1,268 @@
-<path d="M36.7189 14.2753C40.8684 19.4013 42.2125 27.3383 37.8878 34.8344C44.9009 29.3777 48.349 19.6218 45.31 9.97611C44.3165 6.8895 40.2255 5.78714 37.5955 7.82651C37.5371 7.82651 37.5371 7.88163 37.4787 7.88163C35.3747 9.48005 35.0825 12.2911 36.7189 14.2753Z" fill="#2B3B4E"/>
-<path d="M18.1927 43.6533C21.5824 38.0313 28.8878 33.8423 37.8295 34.8344C29.8229 30.8108 18.8941 31.5273 10.6536 37.921C7.96524 39.9604 8.49122 43.984 11.5303 45.4722C11.5887 45.4722 11.5887 45.5273 11.6471 45.5273C13.9849 46.6848 16.8486 45.8029 18.1927 43.6533Z" fill="#2B3B4E"/>
-<path d="M55.3623 47.0153C48.466 46.7949 41.1022 42.7712 37.771 34.8342C37.8294 43.3775 43.6153 52.1413 53.4921 55.9996C56.7065 57.2673 60.1546 54.9523 60.1546 51.7004C60.1546 51.6452 60.1546 51.6452 60.1546 51.5901C60.1546 49.1098 57.9922 47.0704 55.3623 47.0153Z" fill="#2B3B4E"/>
-<path d="M37.8291 40.8979C41.3796 40.8979 44.2578 38.1835 44.2578 34.835C44.2578 31.4865 41.3796 28.772 37.8291 28.772C34.2786 28.772 31.4004 31.4865 31.4004 34.835C31.4004 38.1835 34.2786 40.8979 37.8291 40.8979Z" fill="white"/>
-<path d="M37.8293 38.8583C40.1855 38.8583 42.0956 37.0568 42.0956 34.8347C42.0956 32.6125 40.1855 30.811 37.8293 30.811C35.4731 30.811 33.563 32.6125 33.563 34.8347C33.563 37.0568 35.4731 38.8583 37.8293 38.8583Z" fill="#E6E9EE"/>
-</svg>
-`;
+const ceilingFansXML = `<svg width="71" height="70" viewBox="0 0 71 70" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
some of these aren't 70x70 still but if it's coming off the figma then I think it's fine

Reply:
Yes it's coming off of the Figma. Bobby even made everything 70x70 on Figma, but then when I copy the XML strings of the SVG badges it gives 71x70. Not sure why

Reply:
ok i think its good then ill approve it